be/src/vec/columns/column.h

```diff
@@ -453,6 +453,7 @@ class IColumn : public COW<IColumn> {
       * For example, if nan_direction_hint == -1 is used by descending sorting, NaNs will be at the end.
       *
       * For non Nullable and non floating point types, nan_direction_hint is ignored.
+      * For array/map/struct types, comparison means equal.
```

Comment:
'comparison means equal' is not accurate.

Reply:
Do you mean none zero return code has no meaning？

Reply:
This note is not accurate. 
My actual expression is
  For array/map/struct types,  we compare with nested column element and offsets size